build.gradle

```diff
@@ -1,11 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+  ext.kotlin_version = '1.0.6'
   repositories {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.1.2'
+    classpath 'com.android.tools.build:gradle:2.2.0'
```

Comment:
It should be gradle 2.1.2 for now. I'm not sure if Travis CI supports Gradle 2.2.0. Changes in build script are not in the scope of this PR.